source/swift.txt

```diff
@@ -123,6 +123,13 @@ Compatibility
 MongoDB Compatibility
 ~~~~~~~~~~~~~~~~~~~~~
 
+The compatibility table in this section specifies the recommended version or
+versions of the MongoDB C driver for use with a specific version of MongoDB.
```

Comment:
I: This sentence mentions the C driver but is on the Swift page
S: double check all copied content to make sure driver matches